shared/src/business/useCases/getPractitionerCasesInteractor.ts

```diff
@@ -30,12 +32,22 @@ export const getPractitionerCasesInteractor = async (
     .getPersistenceGateway()
     .getCasesByDocketNumbers({ applicationContext, docketNumbers });
 
-  cases.forEach(
-    aCase => (aCase.caseTitle = Case.getCaseTitle(aCase.caseCaption)),
```

Comment:
`formatCase` below takes care of this. Rather than call it once in one place (the PractitionerCaseList) and not in another (the PDF of the case list), I decided to keep it consistent.